openfl/text/TextField.hx

```diff
@@ -2082,13 +2106,9 @@ class TextField extends InteractiveObject {
 		switch (key) {
 			
 			case RETURN, NUMPAD_ENTER:
-				
-				if (__textEngine.multiline) {
-					
-					replaceSelectedText ("\n");
-					dispatchEvent (new Event (Event.CHANGE, true));
-					
-				}
```

Comment:
@Greg209: nice PR, but why remove the `if (__textEngine.multiline) {` block here?